jcli/src/jcli_app/certificate/new_vote_plan.rs

```diff
@@ -0,0 +1,36 @@
+use crate::jcli_app::certificate::{write_cert, Error};
+use chain_impl_mockchain::block::BlockDate;
+use chain_impl_mockchain::certificate::{Certificate, Proposals, VotePlan};
+use std::ops::Deref;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+pub struct VotePlanRegistration {
+    #[structopt(short = "vs", long = "vote-start")]
+    pub vote_start: BlockDate,
+
+    #[structopt(short = "ve", long = "vote-end")]
+    pub vote_end: BlockDate,
+
+    #[structopt(short = "ce", long = "committee-end")]
+    pub committee_end: BlockDate,
```

Comment:
@NicolasDP , is it correct that we should take this information from the CLI? or should I take it from somewhere else?

Reply:
take what from somewhere else? The blockdate?

Reply:
Brainfart. I have it clear now.